include/dem/particle_wall_contact_info_struct.h

```diff
@@ -32,17 +32,59 @@
 using namespace dealii;
 
 template <int dim>
-struct particle_wall_contact_info_struct
+class particle_wall_contact_info
```

Comment:
You forgot to change the name of the .h and .cc :)

Reply:
Good catch, fixed :)